1_Get_Started/practice/closure_nkang.js

```diff
@@ -0,0 +1,32 @@
+function range(start,end) {
+	var arr = [];
+	if (end === undefined) {
+		return function addPoint(end) {
+			var arr = [];
+			for (let i = start; i <= end; i++) {
+				arr.push(i);
+			}
+			return arr;
+		}
+	}
+	else {
+		var arr = [];
```

Comment:
블록 내부에서만 한정해서 쓰이는 변수라면 var 보다 let을 사용하는게 가독성이 저는 좋아보여서 리뷰했습니다!

함수 안에서
함수 전체 범위: var
블록 범위: let, const

```suggestion
		let arr = [];
```

Reply:
넵 감사합니다.